docs/dev_guide/new_objects.rst

```diff
@@ -4,6 +4,37 @@
 Creating new SunPy Subclasses (Maps, TimeSeries)
 ************************************************
 
+Documenting new data sources
+----------------------------
+Each subclass of `~sunpy.map.GenericMap` or `~sunpy.timeseries.GenericTimeSeries`
+must provide a detailed docstring providing an overview of the data source that
+the object represents. In order to maintain consistency and completeness the
+following information must be provided by a data source docstring, if available,
+and preferably in the following order:
+
+* the name of the mission and instrument and the institution that built it
+* a short description of the instrument (e.g. Cassegrain reflector, Wolter-1 grazing incidence x-ray, coronagraph) including the type of detector
+* a description of the platform (e.g. satellite in 28 deg inclined orbit, a telescope on the summit of Mauna Kea in Hawaii)
+* a description of the primary purpose or science goals of the instrument.
+* a list of all wavelength(s) or passbands in appropriate units
+* a description of the emission processes which dominate in those passbands
+* the field of view and resolution (e.g. angular resolution)
+* the measurement cadence
+* a description of the operational concept (e.g. operates 24/7, observes from 7 am to 5 pm UT)
+* the start and end of the data set
```

Comment:
It might be also good to have a caveats section, such as 'be warned, this instrument looks off at the crab nebula every second tuesday' or the fact that it only supports a certain data product level or things of note along those lines for instrument specific cases.

Reply:
I was thinking that that would be part of the operational concept section. I'll add text to make this more clear.